requirements.txt

```diff
@@ -1,11 +1,23 @@
-# The "base" version of the Mkdocs project.
-# https://github.com/mkdocs/mkdocs
+# The following are something akin to peer dependencies. They are represented
+# as ranges in order to support interoperability with other mkdocs plugins or
+# packages that might otherwise exist in an adopter's environment.
 mkdocs>=1.2.2
+Markdown>=3.2,<3.4
+
+# The following are more akin to direct dependencies. Each line represents one
+# or more features that are provided by `techdocs-core`, and thus are always
+# pinned to an exact version. Bumps should be accompanied by release notes
+# explaining what was added or fixed (or at least pointing to the underlying
+# release notes of the bumped package).
 mkdocs-material==8.1.11
-mkdocs-monorepo-plugin~=1.0.1
+mkdocs-monorepo-plugin==1.0.3
 plantuml-markdown==3.5.1
-markdown_inline_graphviz_extension>=1.1.1,<2.0
+markdown_inline_graphviz_extension==1.1.1
```

Comment:
I see that `markdown_inline_graphviz_extension>=1.1.1,<2.0` was introduced as part of this PR https://github.com/backstage/mkdocs-techdocs-core/pull/74 after this comment https://github.com/backstage/mkdocs-techdocs-core/pull/74#pullrequestreview-1055171341 

> @iswariyam I talked to @iamEAP and we're a little concerned about the open-endedness of themarkdown_inline_graphviz_extension version constraint...
> How about combining what we did with the markdown constraint >=1.1.1,<2 or something?

Before we merge this I just want to make sure this is not still a concern?

Reply:
I _think_ moving the dependency from an extract version to a range was mistaken advice on my part in that PR.  The hope is that splitting out the dependencies into these categories (as done in this PR) will help our future selves reason about ranges vs. exact versions.

Reply:
Yes I agree with that, just wanted to double check with you both around that specific comment!  So lets 🚢 it!